todo_app/data/session_items.py

```diff
@@ -67,3 +67,12 @@ def save_item(item):
     session['items'] = updated_items
 
     return item
+
+def delete_item(id):
+    existing_items = get_items()
+    try:
+        existing_items.pop(id)
```

Comment:
You need a catch block if you want to handle the error when the id is wrong. But really this doesn't need to be inside a try and correctly handling errors is outside the scope of this exercise.

If you did want to handle errors, the function in app.py could put delete_item inside a try block and display a message like "Could not delete item. Please enter a valid ID" from the `catch` block.